admin/classes/domain/Resource.php

```diff
@@ -66,19 +78,61 @@ public function getParentResource(){
 
 		//need to do this since it could be that there's only one request and this is how the dbservice returns result
 		if (isset($result['resourceID'])){
-			$object = new ResourceRelationship(new NamedArguments(array('primaryKey' => $result['resourceRelationshipID'])));
+			$object = new Resource(new NamedArguments(array('primaryKey' => $result['resourceID'])));
+			array_push($objects, $object);
+		}else{
+			foreach ($result as $row) {
+				$object = new Resource(new NamedArguments(array('primaryKey' => $row['resourceID'])));
+				array_push($objects, $object);
+			}
 		}
 
-		return $object;
+		return $objects;
+	}
+
+  public function getIsbnOrIssn() {
+    $query = "SELECT *
+      FROM IsbnOrIssn
+      WHERE resourceID = '" . $this->resourceID . "'
+      ORDER BY 1";
+
+		$result = $this->db->processQuery($query, 'assoc');
+
+		$objects = array();
+
+		//need to do this since it could be that there's only one request and this is how the dbservice returns result
+		if (isset($result['isbnOrIssnID'])){
+			$object = new IsbnOrIssn(new NamedArguments(array('primaryKey' => $result['isbnOrIssnID'])));
+			array_push($objects, $object);
+		} else {
+      foreach ($result as $row) {
+				$object = new IsbnOrIssn(new NamedArguments(array('primaryKey' => $row['isbnOrIssnID'])));
+				array_push($objects, $object);
+			}
+
+    }
+
+		return $objects;
+
+  }
+
+	//returns array of parent resource objects
+	public function getParentResources(){
+    return $this->getRelatedResources('resourceID');
 	}
 
 
-	//returns array of related resource objects
+	//returns array of child resource objects
 	public function getChildResources(){
+    return $this->getRelatedResources('relatedResourceID');
+	}
+  
+  // return array of related resource objects
+  private function getRelatedResources($key) {
 
-		$query = "SELECT *
+    $query = "SELECT *
 			FROM ResourceRelationship
-			WHERE relatedResourceID = '" . $this->resourceID . "'
+			WHERE $key = '" . $this->resourceID . "'
```

Comment:
About multiple parents resources: I think that will be useful with gokb : packages will be a resource linked to multiple resources.